[Whoops] Mother Russia Bleeds

RandomMarine opened the pull request:
:cl: RandomMarine
add: The Russians are now expanding onto the shuttle business. A new escape shuttle is available for purchase.
/:cl:

Reasoning: I feel we need more purchasable shuttles with relatively tame gimmicks to balance out all the crazier ones like luxury, wabbajack, snappop, etc. I think a shuttle with built-in fight pit fits right in alongside those like the escape bar. Plus, who doesn't want more valid reasons get into fights in the final minutes of the round?

One button in the cockpit opens/closes the shutters between it and the arena.
The other button releases bears into the arena. They don't break stuff in the way of targets, so they won't get out unless someone lets them.
![shuttlefinal](https://cloud.githubusercontent.com/assets/6583225/22821337/6cdcf3fc-ef2f-11e6-9a91-041ba5bf94f9.png)

RandomMarine committed:
adds rage cage shuttle

ExcessiveUseOfCobblestone commented:
make it so you have to go through the rage cage to get to the 'safe' area of the shuttle

RandomMarine commented:
>relatively tame gimmicks to balance out all the crazier ones

Cyberboss commented:
Line it with meatspikes

KorPhaeron commented:
@ExcessiveUseOfCobblestone 

Deathgrilles aren't any good for the cage now that they'd tesla everyone and end the fight

FantasticFwoosh commented:
The ship should be named "Mother Russia Bleeds", swap out all the walls for rusty ones, put some Russkie things around and you have perfect theming.

WJohn commented:
This shuttle looks pretty hideous, but the idea is nice. Please rework the design, it's weirdly asymmetrical.

RandomMarine commented:
The sec area was deformed because I originally had the cage right in the middle, so I cut out a corner to allow passage between it and the sec room, and extended it a little to compensate. Then I moved the cage aside and forgot to undo that.

The cargo area's change is a necessity so that the door connects with the docking bay. I can similarly extend the medical area, but I'm worried that I'm providing too much extra space already. But then again, with all the fighting, the medical room will need it.

I'll think about any special theming after settling on a layout. 

Here's how it looks in-game with smoothing currently.
Changes from the OP version.
- Weirdly elongated security room undone.
- Med and cargo areas made symmetrical, prettier, but larger.
- Med room's burn kit replaced with brute kit.
- Added a couple tables around the pit itself with a stack of bruise packs each. Y'know, for healing booboos and tabling scrubs.
![shuttle](https://cloud.githubusercontent.com/assets/6583225/22748048/ab482f56-eddd-11e6-84e9-3bc0875d9a95.png)

coiax commented:
Rapid ideas, disregard any or every:

- Needs cleaning products near the cage
- Needs body bags near the cage

- Change the Cargo and Medbay sections to the RED CORNER and BLUE CORNER, each with their own care facilities (medbay, sleeper), wardrobe, morgue tray.

- Needs a showers/gym/laundry room, with bottles of spray tan

- Maybe put built in flashers inside the pit for subduing of the fighters
- People need signs and foam fingers, and also refreshments
- Referee Drone, one with a black and white hat or something, equipment include space cleaner, body bags, megaphones
- Possibly make the cockpit/command section overlook the brawl pit, the "finest seats in the house", have the captain seat at the front so he can give a thumbs up or down to the gladiators
- Lions

Armhulen commented:
make this shuttle take longer. TWO MINUTES is nothing

RandomMarine commented:
@coiax I could make it a russain fight pit with a button in the cockpit to release bears but it will just become one of those things that no non-antag could press without getting banned.

RandomMarine committed:
mother russia bleeds instead

coiax commented:
Just fyi, currently "in custody" fires if you're on the red shuttle plasteel tiles. We should eventually change it to an area on the shuttle, rather than floor tiles, but currently a huge chunk of this shuttle is redtext giving. Which is hilarious.

RemieRichards commented:
@coiax actually no, they check for:
```/turf/open/floor/plasteel/shuttle/red```
or
```/turf/open/floor/mineral/plastitanium/brig```

While this shuttle uses
```/turf/open/floor/mineral/plastitanium```

Niknakflak commented:
rekt

coiax commented:
topkek

coiax commented:
@GunHog also, having ALL the walls rusty is a bit weird. Try to spread a few non-rusty walls in, otherwise it looks... silly?

GunHog commented:
Why are you asking me this? :confused:

coiax commented:
@RandomMarine also, having ALL the walls rusty is a bit weird. Try to spread a few non-rusty walls in, otherwise it looks... silly?

RandomMarine committed:
adds mother russia bleeds shuttle

Jordie0608 commented:
`_maps/shuttles/emergency_russiafightpit.dmm:"ar" = (/obj/machinery/button/door{id = "ohnorevs"; name = "window shutters"; pixel_x = 26; pixel_y = -6; req_access_txt = "19"; step_x = 0; step_y = 0},/obj/structure/chair,/obj/machinery/button/door{id = "bearsbearsbears"; name = "release bears"; pixel_x = 26; pixel_y = 6; req_access_txt = "20"},/turf/open/floor/mineral/titanium/blue,/area/shuttle/escape)`

Remove `step_x` and `step_y` from the this https://github.com/tgstation/tgstation/pull/23844/files#diff-9d58aae8c6e225f6c6ae4f9d143f4471R18

RandomMarine committed:
Adds mother russia bleeds shuttle

RandomMarine commented:
Also shit. I forgot to consider that the hammer and sickle may not rotate on a horizontal dock like in metastation.

I also forgot to put a status screen near the cockpit.

Kromgar commented:
IT HAS NO OXYGEN

Kromgar commented:
@RandomMarine  NO OXYGEN

Armhulen commented:
Mother Russia breaths?

RandomMarine commented:
That shouldn't happen. Wasn't happening on tests. @Kromgar gonna need specifics. What map?

RandomMarine commented:
So turns out the air issue is unrelated.

Still, this leaves the issue that the hammer and sickle still gets fucked up when the shuttle docks on a map that rotates it.

Cyberboss commented:
Hi air issue was my fault. Sorry, mkay 🤗

On Feb 10, 2017 12:23 PM, "RandomMarine" <notifications@github.com> wrote:

> So turns out the air issue is unrelated.
>
> Still, this leaves the issue that the hammer and sickle still gets fucked
> up when the shuttle docks on a map that rotates it.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/tgstation/tgstation/pull/23844#issuecomment-279006436>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHywen8lIKFkd3eUeKS-sO3CQJ-z4FzJks5rbJ0LgaJpZM4L6T8M>
> .
>

RandomMarine commented:
Seeing as I both have to remove modified files and resolve conflicts, I'll be less hoops to jump through to just open another PR from a new branch.

Russia iz not ded yet comrades
